src/scripts/useredit.js

```diff
@@ -21,11 +21,23 @@ define(["jQuery", "loading", "libraryMenu", "fnchecked"], function($, loading, l
         })
     }
 
+    function loadPasswordResetProviders(page, user, providers) {
+        providers.length > 1 && !user.Policy.IsAdministrator ? page.querySelector(".fldSelectPasswordResetProvider").classList.remove("hide") : page.querySelector(".fldSelectPasswordResetProvider").classList.add("hide");
+        var currentProviderId = user.Policy.PasswordResetProviderId;
+        page.querySelector(".selectPasswordResetProvider").innerHTML = providers.map(function(provider) {
+            var selected = provider.Id === currentProviderId || providers.length < 2 ? " selected" : "";
```

Comment:
```suggestion
            var selected = (provider.Id === currentProviderId || providers.length < 2) ? " selected" : "";
```
IMHO better to add explicit braces for the sake of clarity.